Makefile

```diff
@@ -11,9 +14,13 @@ clean:
 	rm -rf node_modules/
 	rm -rf app/node_modules/
 
-build:
-	make ansible
-	vagrant up --provision
+docker-build: ## Builds Docker image for creating Sunder Linux deb packages.
+	docker build . -t sunder-build
+
+build: docker-build ## Builds Sunder Debian packages for Linux.
+	docker run -v "$(PWD):/sunder" \
```

Comment:
I don't recommend bind mounting the entire code directory in the container. There is at least one serious problem with it, which I encountered while working on my own branch to implement Docker builds: `node_modules` cannot be shared between the host and container if the host and container are not running the same OS/arch (e.g. my macOS development host vs. a Linux build container), due to our project's use of native modules.

Reply:
Blacklisting dirs via `.dockerignore` may be a sane approach here, to ensure that e.g. node_modules aren't included.

Reply:
the ignore file only works with context during build time.. not with volume mounting unfortunately. Realistically we should have a  docker container for node instead of asking devs to run raw commands but thats another scope. So I say we do `COPY` in docker build with the ignore logic to get around this... i tried a couple other solutions but havent found a work-around yet.

someone suggested just running `-v /sunder/node_modules` which mounts a blank directory there but its root owned :|

Reply:
@msheiny I'd recommend `COPY`ing the source into the container, in which case `.dockerignore` will work for you. I think the only directory you should need to bind mount is the directory where the build output ends up.